docs/client_advanced.rst

```diff
@@ -424,7 +424,15 @@ same thing as the previous example, but add another call to
                              '/path/to/client/private/device.key')
   r = await session.get('https://example.com', ssl=sslcontext)
 
-There is explicit errors when ssl verification fails
+If you need to use client-side SSL certificates for authentication
+you can create the :class:`ssl.SSLContext` with that purpose::
+
+  sslcontext = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
```

Comment:
Should it be `ssl.Purpose.SERVER_AUTH`?
https://docs.python.org/3/library/ssl.html#ssl.Purpose.SERVER_AUTH

Reply:
Here's a good article that describes my use case:

https://medium.com/@sevcsik/authentication-using-https-client-certificates-3c9d270e8326

TL;DR: The server authenticates the client during the SSL handshake by using the client's cert. This is different from the "normal" flow where the client checks the server's cert. In fact, in a development environment we have required client certs that are used to talk to servers with self-signed certs so the *only* check is that the client sends a valid cert that the server is preconfigured to accept.

Reply:
Got you. The problem is: you are trying to describe a working and useful but not common use case.
I afraid added 10 lines produce more confusion than help.
A much more comprehensive text is needed to describe a motivation for the mode, like the mentioned article.

But I have a feeling that aiohttp doc is not a good place for such articles, the library *uses* `ssl`, describes very trivial cases, but don't digs into SSL too deep -- the subject is out of `aiohttp` scope (like basic Python programming and many other things).